doc/docs/tutorials/test_servers.md

```diff
@@ -45,10 +45,10 @@ There are *3 Docker server* configurations you can use for testing:
     to the FHIR server that you brought up using the [Synthea data
     uploader](https://github.com/google/fhir-data-pipes/blob/master/synthea-hiv/README.md#Uploader).
 
-    The uploader requires the `google-auth` Python library, which you can install using:
+    The uploader requires that you install the `uploader` module requirements first. You can do this by running:
```

Comment:
Please also add a note about using `virtualenv` as the procedure here can impact the system-wide Python installation. For an example, take a look at [this section](https://github.com/google/fhir-data-pipes/tree/master/dwh#setting-up-apache-spark).